packages/noise/package.json

```diff
@@ -55,5 +54,6 @@
   "peerDependencies": {
     "@flatten-js/interval-tree": "^1.0.19"
   },
-  "gitHead": "914a1e0ae4a573d685841ca2ea921435057deb8f"
+  "gitHead": "914a1e0ae4a573d685841ca2ea921435057deb8f",
+  "module": "dist/index.js"
```

Comment:
is this change intended? what is it used for?

Reply:
I see, would it make sense to leave that this change for a follow up where we align it everywhere? Might make sense to bundle it with the `nx` followup since it's related to how we handle local changes during development (eg pointing to the ts source vs having a build watcher)

Reply:
> is this change intended? what is it used for?
I think @roninjin10 might have added this? I can't recall needing this

> if we use those fields we should probably standardize their order/position in package.json
100%! I've now twice tried to add [package.json eslint plugin](https://github.com/zetlen/eslint-plugin-package-json) to get consistent ordering in our package.json keys but it fails with a parse error as if it can't parse JSON.

Reply:
reverted the `module` change and added a follow up issue for the package.json order: https://github.com/latticexyz/mud/issues/617